.automation/build-srpm.sh

```diff
@@ -11,12 +11,6 @@ ENGINE_VERSION="master"
 # Additional dependencies, which are going to be added to engine and which need
 # to be included in ovirt-engine-build-dependencies, so proper build can pass
 ADDITIONAL_DEPENDENCIES="
-org.ovirt.engine.api:metamodel-server:1.3.10
-org.ovirt.engine.api:metamodel-runtime:1.3.10
-org.ovirt.engine.api:metamodel-tool:1.3.10
-org.ovirt.engine.api:model:4.5.12
-org.ovirt.engine.api:model:4.5.12:jar:sources
-org.ovirt.engine.api:model:4.5.12:jar:javadoc
```

Comment:
Is this removal intended?

Reply:
Yes, in the meantime it already became part of engine, so they will be fetched automatically from engine sources: https://github.com/oVirt/ovirt-engine/pull/687
We need to add them manually here only to be able to build engine which introduces new or updated java dependencies